src/components/avatar/avatar.ts

```diff
@@ -0,0 +1,76 @@
+import { html, LitElement } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { Constructor } from '../common/mixins/constructor.js';
+import { SizableMixin } from '../common/mixins/sizable.js';
+import { styles } from './avatar.material.css';
+
+export class IgcAvatarComponent extends SizableMixin(
+  <Constructor<LitElement>>LitElement
+) {
```

Comment:
This could be simplified.
```suggestion
export class IgcAvatarComponent extends SizableMixin(LitElement) {
```